pkg/labels/array.go

```diff
@@ -168,9 +179,13 @@ func (ls LabelArray) Has(key string) bool {
 // ["cidr:1.1.1.1/32"].Has("cidr.1.0.0.0/8") => true
 // ["cidr:1.0.0.0/8"].Has("cidr.1.1.1.1/32") => false
 func (ls LabelArray) Get(key string) string {
-	keyLabel := parseSelectLabel(key, '.')
+	keyLabel := ParseSelectDotLabel(key)
```

Comment:
Seems we could `dry` this up and leverage lookupLabel?

Reply:
`Get` is an implementation of an upstream k8s interface, but yes, the implementation can be changed :-)